docs/usage.md

```diff
@@ -277,33 +321,52 @@ The time series stack output can be enabled using:
 
 ### Visualization
 
-The workflow provides two types of results visualization and aggregation. The fine grained mosaic visualization contains all time series analyses results for all tiles in the original resolution. Pyramid visualizations present a broad overview of the same data but at a lower resolution. Both visualizations can be enabled or disabled using the parameters `mosaic_visualization` and `pyramid_visualization`. By default, both visualization methods are enabled. Note that the mosaic visualization is required to be enabled when using the `test` and `test_full` profiles to allow the pipeline to check the correctness of its results (this is the default behavior, make sure to not disable mosaic when using test profiles) .
+The workflow provides two types of results visualization and aggregation.
+The fine grained mosaic visualization contains all time series analyses results for all tiles in the original resolution.
+Pyramid visualizations present a broad overview of the same data but at a lower resolution.
+Both visualizations can be enabled or disabled using the parameters `--mosaic_visualization` and `--pyramid_visualization`.
+By default, both visualization methods are enabled.
+Note that the mosaic visualization is required to be enabled when using the `test` and `test_full` profiles to allow the pipeline to check the correctness of its results.
 
 The visualizations can be enabled using:
 
 ```bash
-mosaic_visualization  = '[boolean]'
-pyramid_visualization = '[boolean]'
+--mosaic_visualization  = '[boolean]'
+--pyramid_visualization = '[boolean]'
 ```
 
-### FORCE configuration
+### Intermediate data publishing
 
-FORCE supports parallel computations. Users can specify the number of threads FORCE can spawn for a single preprocessing, or higher level processing process. This is archived through the `force_threads` parameter.
+By default, preprocessing and higher level processing steps do not publish the `.tif` files that they generate to avoid bloating the available storage.
+However, analysis-ready-data (aka. ARD or level-2 data) and the results of time series analyses (aka. level-3 data) maybe be interesting to certain users.
+The files for preprocessing and higher level processing can, therefore, be published by setting the `--save_ard` and `--save_tsa` to `true`, respectively.
 
-The number of threads can be passed using:
+Publishing of intermediate data can be enabled using:
 
 ```bash
---force_threads '[integer]'
+--save_ard  = '[boolean]'
+--save_tsa  = '[boolean]'
 ```
 
-The default value is 2.
+### Module output publishing
+
+Certain users may not be interested in the pipeline's outputs.
```

Comment:
why would one want to run a pipeline, while not being interested in the outputs? 🤔

Reply:
This was requested by a colleague for evaluating workflow execution approaches using different task placement strategies.